manifests/client.pp

```diff
@@ -61,6 +63,8 @@
   Boolean $pam                                         = false,
   String $cipher                                       = 'AES-256-GCM',
   String $tls_cipher                                   = 'TLS-DHE-RSA-WITH-AES-256-GCM-SHA384:TLS-DHE-RSA-WITH-AES-256-CBC-SHA256:TLS-DHE-RSA-WITH-AES-128-GCM-SHA256:TLS-DHE-RSA-WITH-AES-128-CBC-SHA256',
+  Optional[String] $data_ciphers                       = 'AES-256-GCM:AES-128-GCM',
```

Comment:
`Optional` with a default value cannot be forced to undef. Since we test the value against `''`, it make sense to only allow `String`:

```suggestion
  String $data_ciphers                                 = 'AES-256-GCM:AES-128-GCM',
```